api/pyproject.toml

```diff
@@ -172,7 +172,7 @@ pydantic = "~2.7.4"
 pydantic_extra_types = "~2.8.1"
 pydantic-settings = "~2.3.3"
 pgvecto-rs = "0.1.4"
-firecrawl-py = "0.0.5"
+firecrawl-py = "0.0.16"
```

Comment:
pls run dev/sync-poetry to sync lock file

Reply:
> pls run dev/sync-poetry to sync lock file

done